entrypoint.sh

```diff
@@ -11,15 +11,25 @@ fi
 config=`echo $configs | head -n 1`
 interface="${config%.*}"
 
+if [[ "$(cat /proc/sys/net/ipv4/conf/all/src_valid_mark)" != "1" ]]; then
+    echo "sysctl net.ipv4.conf.all.src_valid_mark=1 is not set" >&2
+    exit 1
+fi
+
+sed -i "s:sysctl -q net.ipv4.conf.all.src_valid_mark=1:echo skipping setting net.ipv4.conf.all.src_valid_mark:" /usr/bin/wg-quick
 wg-quick up $interface
 
 docker_network="$(ip -o addr show dev eth0 | awk '$3 == "inet" {print $4}')"
 docker_network_rule=$([ ! -z "$docker_network" ] && echo "! -d $docker_network" || echo "")
 iptables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker_network_rule -j REJECT
 
 docker6_network="$(ip -o addr show dev eth0 | awk '$3 == "inet6" {print $4}')"
-docker6_network_rule=$([ ! -z "$docker6_network" ] && echo "! -d $docker6_network" || echo "")
-ip6tables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker6_network_rule -j REJECT
+if [[ -z "${docker6_network}" ]]; then
+    echo "skipping ipv6 killswitch setup since ipv6 interface was not found..." >&2
```

Comment:
Mind starting the output with a capital letter to be consistent with the style on line 7?

```suggestion
    echo "Skipping ipv6 killswitch setup since ipv6 interface was not found" >&2
```

Or optionally change line 7 to be consistent with this one. Either way 😄

Reply:
Done.